cli/src/send_tpu.rs

```diff
@@ -0,0 +1,31 @@
+use log::*;
+use solana_client::{rpc_client::RpcClient, rpc_response::RpcLeaderSchedule};
+use std::net::{SocketAddr, UdpSocket};
+
+pub fn get_leader_tpu(
+    rpc_client: &RpcClient,
+    slot_index: u64,
+    leader_schedule: &Option<RpcLeaderSchedule>,
```

Comment:
NIT: Could `Option<&RpcLeaderSchedule>`  here, then `as_ref()` at the call site.